app/src/main/java/com/babylon/wallet/android/presentation/account/createaccount/CreateAccountViewModel.kt

```diff
@@ -108,7 +109,9 @@ class CreateAccountViewModel @Inject constructor(
         keystoreManager.resetMnemonicKeySpecWhenInvalidated()
 
         return runCatching {
-            sargonOs.newWallet()
+            sargonOs.newWallet(
```

Comment:
hmm I am not sure this is the cause, but now I you create a new wallet you get the biometric prompt twice for the first account creation.

Reply:
Good catch!